src/xfinity.ts

```diff
@@ -145,23 +149,44 @@ export class Xfinity extends EventEmitter {
     private async authenticate() {
         console.info(`Loading (${LOGIN_URL})`);
         const page = await this.getPage();
-        await page.goto(LOGIN_URL, { waitUntil: 'networkidle0' });
+
+        try {
+            await page.goto(LOGIN_URL, { waitUntil: 'networkidle0' });
+            await page.waitForSelector('title');
+            
+            let pageTitle = await page.title();
+            console.log('Page Title: ', pageTitle);
+            if (pageTitle == LOGIN_TITLE) {
```

Comment:
```suggestion
            if (pageTitle === LOGIN_TITLE) {
```

Linting should fix this.

Reply:
It did.